tests/testthat/test-rename.r

```diff
@@ -1,4 +1,7 @@
-context("Rename")
+#################################################
```

Comment:
I'm sorry, I don't understand.  I don't see any examples like that in the package (even when I replace the forward slashes with pound signs).  Do you mean change
```r
#################################################
### Duplicate Names
#################################################
```

to
```r
## Duplicate Names -------------------
```
?